modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,165 @@
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, enter `Scheduler`.
+. Select *Scheduler*.
+. In the *Scheduling Strategy* field, select whether you want to use a fixed frequency or cron expression to trigger the flow:
++
+For example, the following configuration directs `Scheduler` to:
+
+* Wait 10 seconds before initiating the flow for the first time after the app starts
+* Initiate the flow every 30 seconds after the initial 10 second delay
++
+.Select `Fixed Frequency` in the Scheduling Strategy field to initiate a flow at regular intervals.
+image::introduction/intro-config-fd-scheduler-fixed.png[Configuration for a fixed scheduling strategy]
++
+
+The following configuration uses a cron expression to initiate an action every second:
++
+.Select `Cron` in the *Scheduling Strategy* field to use a cron expression to specify when to initiate a flow.
+image::introduction/intro-config-fd-scheduler-cron.png[Configuration using a cron expression]
+
+For more information about configuring `Scheduler`, see xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint]
+
+
+[[connector-specific-trigger]]
+=== Use a Connector-Specific Trigger
+
+If a connector has associated triggers, you can use a connector-specific trigger to initiate the flow. For example, if you are configuring an app that contains an Amazon SQS Connector operation, you can use the `Amazon SQS Receive Messages` trigger to initiate a flow each time the configured Amazon SQS queue receives a new message.
+
+To start the configuration for a connector-specific trigger, follow these steps:
+
+. Follow the procedure described in <<configure-trigger,Configure the Trigger For the Flow>>.
+. In the *Search* field, search for and select the connector.
+. If the connector has multiple triggers, select the trigger to use. (If the connector has one trigger, that trigger is selected automatically.)
+
+Each connector-specific trigger has a unique configuration, but most configurations require the following actions:
+
+* Completing the required fields.
+* Selecting an existing connection or configuring a new one.
+
+For more information about configuring a connector-specific trigger, see the reference guide for the associated connector.
+
+== Configure the Target Component
+
+After you configure a trigger for a connector, you can configure the connector as the target component:
+
+NOTE: Depending on the connector you are working with, the step order in the following procedure might vary.
+
+. Click the *+* icon next to the trigger card.
++
+.Clicking the *+* icon brings up a screen where you can select the target connector operation.
+image::introduction/intro-config-fd-plus-target.png[Click the Plus Sign]
++
+. In the *Select a component* screen, search for and select the target connector.
+. Select the target operation.
+. Select an existing connection configuration or add a new one.
+. To add a new connection:
```

Comment:
```suggestion
To add a new connection:
```
But, once again, watch the subtasks. Remember that THIS topic is about configuring the target component, not about adding connections, a separate task that we might possibly assume that our audience is familiar with and that optimally appears in some other location that we could link to, if really necessary. Your choice; just sayin'....